scripts/release/README.md

```diff
@@ -1,27 +1,31 @@
 ## How to release widgets & modules to the appstore (`./marketplaceRelease.js`)
 
 #### Web Widgets
-1. Create a release in Github
```

Comment:
For web widgets we still need to create a github release and manually add the mpk to the tag..